app/home/translate/lib/openai.ts

```diff
@@ -58,3 +58,17 @@ export const sendMessageToOpenAI = async (
     throw error;
   }
 }
+
+export const validateOpenAIKey = async (key: string): Promise<boolean> => {
+  const openai = new OpenAI({ apiKey: key, dangerouslyAllowBrowser: true })
+  try {
+    const stream = await openai.chat.completions.create({
+      model: 'gpt-3.5-turbo',
+      messages: [{ role: 'user', content: 'Say this is a test' }],
+      stream: true,
+    });
+    return true;
+  } catch {
```

Comment:
shouldn't this be `} catch (error) {` ?

Reply:
We can ignore the params because we are not going to use it.